modules/objdetect/include/opencv2/objdetect.hpp

```diff
@@ -304,7 +305,8 @@ class CV_EXPORTS_W CascadeClassifier
                                   int minNeighbors = 3, int flags = 0,
                                   Size minSize = Size(),
                                   Size maxSize = Size(),
-                                  bool outputRejectLevels = false );
+                                  bool outputRejectLevels = false,
+                                  int rejectStage = 3);
```

Comment:
I would put the default value at 0, not 3. It is not the expected behaviour to output detections of the last 3 stages ... not if you do not want this... So why not set it to 0?